src/methods/sep31Send/sendPayment.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Asset,
+  BASE_FEE,
+  Keypair,
+  Memo,
+  Operation,
+  Server,
+  TransactionBuilder,
+} from "stellar-sdk";
+import { log } from "helpers/log";
+
+interface SendPaymentProps {
+  secretKey: string;
+  assetCode: string;
+  assetIssuer: string;
+  networkUrl: string;
+  networkPassphrase: string;
+  amount: string;
+  sendMemo: string;
+  sendMemoType: string;
+  receiverAddress: string;
+}
+
+export const sendPayment = async ({
+  secretKey,
+  assetCode,
+  assetIssuer,
+  networkUrl,
+  networkPassphrase,
+  amount,
+  sendMemo,
+  sendMemoType,
+  receiverAddress,
+}: SendPaymentProps) => {
+  log.instruction({ title: "Send stellar payment to receiving anchor" });
+
+  const keypair = Keypair.fromSecret(secretKey);
+  const server = new Server(networkUrl);
+  const asset = new Asset(assetCode, assetIssuer);
+  const account = await server.loadAccount(keypair.publicKey());
+
+  const accountBalance = account.balances.find(
+    (b: any) =>
+      b.asset_code === asset.getCode() && b.asset_issuer === asset.getIssuer(),
+  );
+
+  if (!accountBalance) {
```

Comment:
I think we can error out and ask the user to 'Trust Asset' and get some tokens before trying to use SEP-31 send, since the balance will be 0 after trust is added anyway.

Reply:
Updated in https://github.com/stellar/stellar-demo-wallet/pull/125